README.md

```diff
@@ -74,9 +77,7 @@ The following is a brief listing of currently supported features:
 
 ## Importing Materials
 
-To import materials from USD, import the USD file using the USD menu. Next, click
-on the root GameObject and select either DisplayColor or Preview Surface from the
-materials dropdown menu.
+To import materials from USD, import the USD file using the USD menu. For the materials to render in the scene, in the Inspector change the Import Settings > Materials to 'Import Preview Surface' and reimport the USD file.
```

Comment:
Very minor suggestion:
To import materials from USD, firstly import the USD file using the USD menu. Then, to get the materials to render in the scene, change the Import Settings > Materials in the Inspector to 'Import Preview Surface'. Finally reimport the USD file.